packages/wdio-allure-reporter/src/index.js

```diff
@@ -110,8 +110,11 @@ class AllureReporter extends WDIOReporter {
         const currentTest = this.allure.getCurrentTest()
 
         if (!this.isMultiremote) {
-            const { browserName, deviceName } = this.capabilities
-            const targetName = browserName || deviceName || cid
+            const { browserName, deviceName, desired } = this.capabilities
+            let targetName = browserName || deviceName || cid
+            if(desired && desired.deviceName && desired.platformVersion){
```

Comment:
minor formatting, also let's add a comment
```suggestion
            /**
             * custom mobile grids can have device information in a `desird` cap
             */
            if (desired && desired.deviceName && desired.platformVersion) {
```